#
# Translation uints are mm, rotation units are degrees
#
std_msgs/Header header
float32 tx
float32 ty
float32 tz
float32 rot_x
float32 rot_y
float32 rot_z
